src/utilities/activityContent.ts

```diff
@@ -0,0 +1,63 @@
+import path from "path";
+import {
+  createVideo,
+  createImage,
+  ActivityContentNote,
+  ActivityContentAttachment,
+} from "@dsnp/sdk/core/activityContent";
+
+export const noteToActivityContentNote = (
+  note: string,
+  uriList: string[]
+): ActivityContentNote => {
+  const activityContent: ActivityContentNote = {
+    "@context": "https://www.w3.org/ns/activitystreams",
+    type: "Note",
+    content: note,
+    published: new Date().getTime().toString(16),
+  };
+  if (uriList) {
+    if (typeof uriList === "string") {
```

Comment:
uriList string is an array of strings how could this happen?

Reply:
Yep. This probably became unnecessary in the translation.